packages/front-end/styles/tailwind.css

```diff
@@ -0,0 +1,3 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
```

Comment:
We should be able to throw these at the top if our existing `global.scss` from what I understand

Reply:
yeah you can. i just moved this out to there own if we plan on getting rid of scss